kitsune/flagit/models.py

```diff
@@ -54,6 +54,11 @@ class FlaggedObject(ModelBase):
     handled = models.DateTimeField(default=datetime.now, db_index=True)
     handled_by = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
 
+    assignee = models.ForeignKey(
+        User, on_delete=models.SET_NULL, null=True, related_name="assigned_flags"
+    )
+    assigned = models.DateTimeField(default=None, null=True)
```

Comment:
nit: this can be confused with a boolean flag. Maybe rename it to something like `assigned_timestamp` or anything else that would clearly indicate that's a `DateTimeField`?